system/include/webgpu/webgpu.h

```diff
@@ -324,8 +324,8 @@ typedef enum WGPUCullMode {
 } WGPUCullMode WGPU_ENUM_ATTRIBUTE;
 
 typedef enum WGPUDeviceLostReason {
-    WGPUDeviceLostReason_Undefined = 0x00000000,
-    WGPUDeviceLostReason_Destroyed = 0x00000001,
+    WGPUDeviceLostReason_Unknown = 0x00000001,
```

Comment:
We need to avoid breaking changes for now, so please add:
`WGPUDeviceLostReason_Undefined = 0x00000001,`
(with the same value 1 as Unknown)

and same elsewhere

Reply:
I have uploaded a new commit, please review it.